src/app/components/files/files.component.html

```diff
@@ -30,7 +30,6 @@
           [sorting]="['name', 'ASC']"
           [imageResolver]="imageResolver"
           [headerFilters]="true"
-          [filterValue]="queryParams"
```

Comment:
why ?

Reply:
in addition there is no unit test

Reply:
@davidcanonieto I saw that was introduced by you. Can you double-check?
https://github.com/Alfresco/alfresco-content-app/commit/51184e5b16433ce6e93c73e28d97775f0b3e2b5e#diff-bda27cbfa7ca0c26a8b8679e0e5ed023R33

Reply:
we should not remove this property, it enables the header filters

Reply:
Never mind, it's added underneath